data_structures/queue_using_two_stacks.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
```

Comment:
Please add a comment block explaining what the program does, or how it works.

Reply:
Sure! Will update everything.

Reply:
Sorry @Panquesito7 for interrupting in btw ; This type of documentation needed @shoniavika #835

Reply:
Thanks, it is better like that, simple and easy.
It has a copyright notice, which shall not be added here, though.

Reply:
Okay, so what do I change next? Do I need to add copyright?

Reply:
Add documentation like in #835, without the copyright notice. 🙂

Reply:
Sure!